packages/pinia/test-dts/mapHelpers.test-d.ts

```diff
@@ -161,6 +165,17 @@ describe('mapHelpers', () => {
           set: (v: 'on' | 'off') => any
         }
       }>(mapWritableState(useSetupStore, ['a']))
+
+      expectTypeOf<{
+        a: {
+          get: () => 'on' | 'off'
+          set: (v: 'on' | 'off') => any
+        }
+        writableUpper: {
+          get: () => string
+          set: (v: 'on' | 'off') => any
+        }
+      }>(mapWritableState(useSetupStore, ['a', 'writableUpper']))
```

Comment:
we can remove the `a`
```suggestion
      expectTypeOf<{
        writableUpper: {
          get: () => string
          set: (v: 'on' | 'off') => any
        }
      }>(mapWritableState(useSetupStore, ['writableUpper']))
```